.github/workflows/integration-tests.yml

```diff
@@ -58,7 +58,7 @@ jobs:
         run: (aqueduct start > $SERVER_LOGS_FILE 2>&1 &)
 
       - name: Install packages needed for testing
-        run: pip3 install nltk matplotlib pytest-xdist
+        run: python3 -m pip install nltk matplotlib pytest-xdist
```

Comment:
was this an issue?

Reply:
I'm worried about changing everything to this because our user documentation doesn't usually say you have to prefix with `python -m`

Reply:
This is a general change to avoid the situation where user’s pip3 and python3 aren’t compatible. I encountered the issue because my pip3 was installing things to usr/local or something like that instead of the conda environment. 

I don't think it would be an issue on the Github Actions so I can change it back to what it was before. I made this change to the Github Actions commands to be consistent to my update to how we call pip in the rest of the codebase.

Reply:
ok - I think its fine leaving it, since we do use `python -m` everywhere in our system. It might be helpful changing our gitbook documentation to help users that have similar incompatibilities? What do you think @cw75

Reply:
Yeah I think in general Python3 -m is good in avoiding the potential inconsistencies. We can change the Gitbook documentation as well. I think we should be consistent to use it everywhere, including the Git actions @eunice-chan

Reply:
Added documentation https://github.com/aqueducthq/gitbook/pull/10